diff --git a/node_modules/node-fetch/src/index.js b/node_modules/node-fetch/src/index.js
index 7c4aee8..468b878 100644
--- a/node_modules/node-fetch/src/index.js
+++ b/node_modules/node-fetch/src/index.js
@@ -62,7 +62,19 @@ export default async function fetch(url, options_) {
 		}
 
 		// Wrap http.request into fetch
-		const send = (parsedURL.protocol === 'https:' ? https : http).request;
+
+		//Изменение
+		//const send = (parsedURL.protocol === 'https:' ? https : http).request;
+		/*const send = (() => {
+			if (parsedURL.protocol === 'http:') {
+				return http.request;
+			}
+			if (parsedURL.protocol === 'https:') {
+				return https.request;
+			}
+			throw new TypeError(`Unsupported protocol: "${parsedURL.protocol}"`);
+		})();*/
+		const send = https.request;
 		const {signal} = request;
 		let response = null;
 
